google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -310,27 +611,76 @@ class Page:
+    _hocr_bounding_box: Optional[str] = dataclasses.field(init=False, default=None)
+
+    def __post_init__(self):
+        """
+        Order of Init
+        Token
+        Line
+        Paragraph,
+        Block
+        """
+
+        self.text = _text_from_layout(
+            self.documentai_object.layout, text=self.document_text
+        )
+        self.page_number = int(self.documentai_object.page_number)
 
-    def __post_init__(self, document_text) -> None:
-        self.text = _text_from_layout(self.documentai_page.layout, text=document_text)
-        self.page_number = int(self.documentai_page.page_number)
-        self.form_fields = [
-            FormField(documentai_formfield=form_field, document_text=document_text)
-            for form_field in self.documentai_page.form_fields
-        ]
-        self.lines = [
-            Line(documentai_line=line, document_text=document_text)
-            for line in self.documentai_page.lines
-        ]
-        self.paragraphs = [
-            Paragraph(documentai_paragraph=paragraph, document_text=document_text)
-            for paragraph in self.documentai_page.paragraphs
-        ]
-        self.blocks = [
-            Block(documentai_block=block, document_text=document_text)
-            for block in self.documentai_page.blocks
-        ]
         self.tables = [
-            Table(documentai_table=table, document_text=document_text)
-            for table in self.documentai_page.tables
+            Table(documentai_object=table, document_text=self.document_text)
+            for table in self.documentai_object.tables
         ]
+
+        self.form_fields = [
+            FormField(documentai_object=form_field, document_text=self.document_text)
+            for form_field in self.documentai_object.form_fields
+        ]
+
+        self.tokens = _get_tokens(tokens=self.documentai_object.tokens, page=self)
+        self.lines = _get_lines(lines=self.documentai_object.lines, page=self)
+        self.paragraphs = _get_paragraphs(
+            paragraphs=self.documentai_object.paragraphs, page=self
+        )
+        self.blocks = _get_blocks(
+            blocks=self.documentai_object.blocks,
+            page=self,
+        )
+
+    def to_hocr(self, title):
+        r"""Exports a string hOCR version of the documentai.Document.Page.
+
+        Args:
+            title (str):
+                Required. The title for ocr_page.
+
+        Returns:
+            str:
+                A string hOCR version of the documentai.Document.Page.
+        """
+        f = ""
+        pidx = self.documentai_object.page_number
+        f += f"<div class='ocr_page' lang='unknown' title='image \"{title}\";{self.hocr_bounding_box}'>"
+        for bidx, block in enumerate(self.blocks):
+            f += f"<span class='ocr_carea' id='block_{pidx}_{bidx}' title='{block.hocr_bounding_box}'>"
+            for paridx, paragraph in enumerate(block.paragraphs):
```

Comment:
Consider calling the children elements' `to_hocr` recursively, but if it not obvious how to easily do that, let's just track it in an internal clean up issue/bug.

Reply:
Let's add it to internal clean up in buganizer.